src/rust/metric-forwarder/src/cloudwatch_send.rs

```diff
@@ -428,7 +428,6 @@ mod tests {
         let as_stat = crate::cloudwatch_logs_parse::parse_log(input)?;
```

Comment:
Could we encode this as an integration test? Commented-out code is a smell.

Reply:
We can just delete it, honestly